SPECS/kernel-hyperv/kernel-hyperv.spec

```diff
@@ -92,6 +92,8 @@ This package contains the 'perf' performance analysis tools for Linux kernel.
 %prep
 %setup -q -n CBL-Mariner-Linux-Kernel-rolling-lts-mariner-%{version}
 %patch0 -p1
+# upstream fix: https://github.com/torvalds/linux/commit/d08c84e01afa7a7eee6badab25d5420fa847f783
+sed -i 's/ PTHREAD_STACK_MIN/ (int)PTHREAD_STACK_MIN/g' tools/perf/builtin-sched.c
```

Comment:
This should be a patch rather than an inline sed command in the spec.

Reply:
Sure, will convert to patch

Reply:
Will fix post merge

Reply:
Converted to patch